packages/tradershub/src/components/TradingAccountCard/TradingAccountCardComponent.tsx

```diff
@@ -9,7 +9,7 @@ type TTradingAccountCardContent = {
 
 export const TradingAccountCardContent = ({ children, title }: TTradingAccountCardContent) => (
     <div className='grow'>
-        <Text bold className='leading-200' size='sm'>
+        <Text className='leading-200' size='sm' weight='bold'>
             {title}
         </Text>
         <Text className='text-[12px] leading-100 w-5/6 lg:w-full'>{children}</Text>
```

Comment:
@sergei-deriv i think we can pass size='xs' here and remove the hardcoded text size

Reply:
@thisyahlen-deriv nice catch, didn't notice it when change the imports.